docs/corpus.markdown

```diff
@@ -1,45 +1,166 @@
+---
+title: Corpus
+---
+
 # The QuantGov Corpus
 
+Within the QuantGov framework, a corpus refers to a set of documents and a
+driver that is able to serve those documents, along with a unique index value.
+
 ## Basic Structure
 
-A corpus represents a set of documents, and implements the corpus driver interface. The root directory for a corpus should contain:
+A corpus represents a set of documents, and implements the corpus driver
+interface. The root directory for a corpus should contain a Python module named
+`driver.py` that defines the QuantGov corpus driver for that corpus.
+Additionally, a corpus can contain a file called `metadata.csv` that contains
+any additional information about individual documents that may be relevant. For
+example, the `metadata.csv` generated by the CFR corpus includes which agency
+and department authored each individual CFR part.
 
--   A `makefile` to manage the workflow of the corpus
--   A module named `driver.py` that implements the corpus driver interface (see below)
--   A subdirectory named `scripts` containing the scripts needed to obtain, organize, and prepare the documents that make up the corpus, as well as to generate any metadata for the corpus.
--   A subdirectory named `data` that holds any intermediate data generated during preparation of the corpus.
+## Starting a New Corpus
 
-One file that each corpus should generate is a csv file in the `data` directory named `metadata.csv`. This file should contain any additional information about individual documents that may be relevant. For example, the `metadata.csv` generated by the CFR corpus includes which agency and department authored each individual CFR part, as well as the restriction and word count for each part.
+The fastest way to start a new corpus is to use the command
```

Comment:
Do we want to add something here about installing the library first?

Reply:
Don't need to do this on this page, since we do it on the front page.